### ISSUE TITLE
Change import of node-static in @hocuspocus/extension-monitor

### ISSUE THREAD

**Issue by iSplasher** [ueberdosis/hocuspocus#487]
**Description**
When trying to use @hocuspocus/extension-monitor in a NextJS project, you encounter this error:
```
 x Legacy octal escape is not permitted in strict mode
     ,-[362:1]
 362 |             // Stream the file to the client
 363 |             fs.createReadStream(file, {
 364 |                 flags: 'r',
 365 |                 mode: 0666,
     :                       ^^^^
 366 |                 start: startByte,
 367 |                 end: startByte + (length ? length - 1 : 0)
 368 |             }).on('data', function (chunk) {
     `----
```

This error stems from the `node-static` package that hasn't been updated in well over 4 years.
A quick search shows that the monitor extension [doesn't even use the package, it only imports it.](https://github.com/ueberdosis/hocuspocus/blob/9c9984d7c7f8ba6cc7e08858d4a7c310410cf506/packages/extension-monitor/src/Dashboard.ts#L5)

I propose removing the package from the extension altogether seeing as it's not being used.

**Steps to reproduce the bug**
1. Setup a simple NextJS project
2. Try to import @hocuspocus/extension-monitor

**Expected behavior**
No error message occurs.

**Environment?**
 - operating system:
 - browser:
 - mobile/desktop:
 - hocuspocus version: 1.0.0

**Comment by iSplasher** [comment]
I'm sorry, I was wrong. The extension does use it.
I also found out that the github repo for node-static has been updated but a new npm version just hasn't been released yet, and that doesn't seem to be happening anytime soon.
I instead tried to force yarn to use the repo version with `yarn set resolution`, which worked.
But it appears that the new changes in node-static include not providing a default export for the package. This means, I'm now encountering this error: 
```
ModuleDependencyError: export 'default' (imported as 'nodeStatic') was not found in 'node-static' (possible exports: Server, mime, version)
```

That means my proposed fix is to instead use a named import:
```
import { Server } from 'node-static'
```

This should work with both the old and new version when it eventually comes out.
